forge-gui/res/cardsfolder/upcoming/cass_hand_of_vengeance.txt

```diff
@@ -0,0 +1,11 @@
+Name:Cass, Hand of Vengeance
+ManaCost:2 R W
+Types:Legendary Creature Human Ranger
+PT:4/3
+K:Vigilance
+T:Mode$ ChangesZone | ValidCard$ Creature.Other+enchanted+YouCtrl,Creature.Other+equipped+YouCtrl,Card.Self+equipped,Card.Self+enchanted | Origin$ Battlefield | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigChangeZone | Whenever NICKNAME or another creature you control dies, if it was enchanted or equipped, return any number of Aura cards that were attached to it from your graveyard to the battlefield attached to target creature, then attach any number of Equipment that were attached to it to that creature.
+SVar:TrigChangeZone:DB$ ChangeZone | ChangeType$ Aura.AttachedTo TriggeredCard,Equipment.AttachedTo TriggeredCard | Origin$ Graveyard | Destination$ Battlefield | RememberChanged$ True | SubAbility$ DBAttach
```

Comment:
The changetype isn't correct

Reply:
well first of it doesn't say to return equipment since that doesn't make sense
and then you're not targeting

Reply:
does the new version work now? it's a tricky LKI situation

Reply:
Not yet, any ideas ?

Reply:
check new version of Rhuk inside #4778

Reply:
Thanks, but having some difficulties with the auras part (equipment works fine)

Reply:
you need to use the same Defined syntax for it

Reply:
Does it needs to be `Defined$ AttachedTo TriggeredCardLKICopy.Aur`a or `ChangeType$ Valid AttachedTo TriggeredCardLKICopy.Aura`

Reply:
I'd say ChooseFromDefined$ makes more sense

Reply:
Can't get this one to work for now. I think it might be best to remove this one from this PR ?

Reply:
looks ok now, I'll check

Reply:
you forgot `Hidden$ True`

Reply:
Ah finally working now. Thanks !